R/schema_GridRange.R

```diff
@@ -0,0 +1,40 @@
+# https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/other#GridRange
```

Comment:
There are ~200 so-called schemas in the Sheets API. `GridRange` is an example.

Once you start writing and updating Sheets, you need to create a lot of bodies, in the REST sense, which was not true while we were only reading. And these bodies must be built up of (often nested) schemas. I have to create some sort of machinery around this.

What I do in this PR would not scale nicely to the full set of schemas, but is that necessary? I don't know.

But if I did something more automated re: class creation and validation, then it scales and, in fact, that would belong in gargle, for reuse across multiple APIs.

I don't know what to do.